Cargo.toml

```diff
@@ -20,6 +20,9 @@ categories = ["rendering::graphics-api"]
 # disabled by default
 log-impl = []
 
+[dependencies]
+num_enum = "0.7.3"
```

Comment:
`num_enum` adds full rust parser and a bunch of other dependencies into the build try. I would rather manually implement TryFrom for one enum we need it for.

Reply:
Okay, I used a conversion function instead to convert u8 to MouseButton type. Is this better?